qutebrowser/browser/browsertab.py

```diff
@@ -1016,7 +1016,15 @@ def _on_load_finished(self, ok: bool) -> None:
             return
 
         sess_manager.save_autosave()
+        self.load_finished.emit(ok)
+
+        if not self.title():
+            self.title_changed.emit(self.url().toDisplayString())
 
+        self.zoom.reapply()
+
+    def _update_load_status(self, ok: bool) -> None:
+        """Update the load status after a page finished loading."""
```

Comment:
Right now it's a bit unclear that this needs to be called explicitly by the implementing class. If, for example, there's another backend, it won't get this update unless we add the same function as webkit. Could you either add a note to this docstring explaining that this needs to be called, or find some way to automatically trigger this so the webkit override isn't needed (and some override passed on webengine).

Reply:
I considered doing it without the override, but this was simpler - I updated the docstring in 20d2ed3757c6c53e1149c8599409be6bedefeb7c.